examples/addr2line.rs

```diff
@@ -203,85 +219,96 @@ fn main() {
 
     let ctx = Context::from_dwarf(dwarf).unwrap();
 
-    let stdin = std::io::stdin();
-    let addrs = matches
-        .values_of("addrs")
-        .map(Addrs::Args)
-        .unwrap_or_else(|| Addrs::Stdin(stdin.lock().lines()));
+    let addrs: Box<dyn Iterator<Item = Option<u64>>> = if opts.addrs.is_empty() {
+        Box::new(
+            std::io::stdin()
+                .lock()
+                .lines()
+                .map(|s| parse_uint_from_hex_string(&s.unwrap()).ok()),
+        )
+    } else {
+        Box::new(opts.addrs.iter().copied())
+    };
 
-    for probe in addrs {
-        if print_addrs {
-            let addr = probe.unwrap_or(0);
-            if llvm {
+    for mprobe in addrs {
+        let probe = match mprobe {
+            Some(probe) => probe,
+            None => {
+                println!("??:0");
```

Comment:
I think we should call `print_loc` here to get the `opts.llvm` handling

Reply:
Done.